src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -40,6 +40,15 @@ public SyndicationFeed Feed
             }
         }
 
+        public Func<string, string, string, string> StringParser { get; set; } = DefaultStringParser;
+
+        public Func<string, UriKind, string, string, Uri> UriParser { get; set; } = DefaultUriParser;
+
+        // Different DateTimeParsers are needed for Atom and Rss so can't set inline
+        public Func<string, string, string, DateTimeOffset> DateTimeParser { get; set; }
+
```

Comment:
This will break anyone deriving from SyndicationFeedFormatter as they will get a NRE when calling DateTimeParser. We need something sensible set.

Reply:
Good point. I changed the method to virtual.

Reply:
This was addressed by the change for `GetDefaultDateTimeParser` below.